### PR TITLE
base-system@g.o: git-2->git-r3

### PR DESCRIPTION
**Pull request by hanetzer**
A lot of these packages don't actually use `git-2`, but do include it in conditional code
based on `${PV}`. The intent here is to remove them so newbies looking for inspiration
from existing ebuilds do not try to use a deprecated eclass.

### CODE REVIEW

File: sys-apps/install-xattr/install-xattr-9999.ebuild [hunk 1]
```diff
@@ -22,7 +22,7 @@ SLOT="0"
 
 src_prepare() {
 	tc-export CC
-	eapply_user
+	default
```
Comment (Whissi) [inline sys-apps/install-xattr/install-xattr-9999.ebuild]: `default` should come first.

Comment (hanetzer) [reply sys-apps/install-xattr/install-xattr-9999.ebuild]: Will fix.

**Comment by hanetzer** [comment]
@Whissi so you want bad examples left behind? The ebuilds are non-live, so the changes made
are no-op anyways and the git-2 inheritance will sit there for quite some time.

**Comment by Whissi** [comment]
Sorry, I am unable to review this. Looks like Github forces me to review all ebuilds at once or nothing. No. Sorry, I don't approve this.

**Comment by floppym** [comment]
I don't like this either.

**Comment by floppym** [comment]
Eh, I guess the updates don't really hurt anything. I merged it.

**Pull request closed without merging**